src/DeVinci_frontend/components/ChatBox.svelte

```diff
@@ -197,17 +220,6 @@
     showToast = false;
   }
 
-
-  function interruptMessageGeneration(): void {
```

Comment:
I think this function title well describes the intention of the stop chat button functionality